CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+### Supported Node.js versions
+
+All the current LTS versions are supported.
+
+### npm scripts
+
+```bash
+# install dependencies
+$ npm i
```

Comment:
I think we are using `yarn`, maybe we should change these commands accordingly

Reply:
I didn't work on this file yet since I'm waiting to see if I have to change something since we're using Lerna instead of our standard flow

Reply:
That makes sense